packages/serverpod/lib/src/relic/routes/static_directory.dart

```diff
@@ -71,7 +72,7 @@ class RouteStaticDirectory extends Route {
       // Enforce strong cache control.
       request.response.headers.set('Cache-Control', 'max-age=31536000');
 
-      var filePath = path.startsWith('/') ? path.substring(1) : path;
+      var filePath = path.subString(1);
```

Comment:
Question: Does this resolve to the same filePath as before? If I understand the change correctly, for the string `"alex/yahu"` the old implementation would return `"alex/yahu"` while the new one would return `"lex/yahu"` right?

Reply:
Yes, sorry I forgot to push the updated code, just gimme some moments i will update it.